userspace/libscap/scap_bpf.c

```diff
@@ -1324,6 +1375,93 @@ int32_t scap_bpf_close(scap_t *handle)
 	return SCAP_SUCCESS;
 }
 
+static int32_t scap_bpf_handle_kt_mask( scap_t *handle, uint32_t op, uint32_t kt_index)
```

Comment:
We are preparing to add uprobe as a new program type in some other plans. Can we abstract k_t to another level. #37

Reply:
uprobe function is not merge yet , now we can merge first